float64[7] joints
---
